jmp-service-rest/src/main/java/com/epam/learning/backendservices/rest/converter/UserToUserResponseDtoConvertor.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.learning.backendservices.rest.converter;
+
+import com.epam.learning.backendservices.rest.dto.UserResponseDto;
+import com.epam.learning.backendservices.rest.model.User;
+import org.springframework.beans.BeanUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserToUserResponseDtoConvertor {
+
+    public UserResponseDto convert(User user) {
```

Comment:
There is interface that can fit in your case [Converter](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/core/convert/converter/Converter.html). Optional: implement interface.

Reply:
Done